src/test/rpc_tests.cpp

```diff
@@ -114,6 +114,7 @@ BOOST_AUTO_TEST_CASE(rpc_rawsign)
     std::string privkey2 = "\"Kyhdf5LuKTRx4ge69ybABsiUAWjVRK4XGxAKk2FQLp2HjGMy87Z4\"";
     NodeContext node;
     node.chain = interfaces::MakeChain(node);
+    node.mempool = std::move(m_node.mempool);
```

Comment:
In commit "node: Use mempool from node context instead of global" (fa963d3818afea82061615b9939361444e5b5c2d)

I think you can get rid of this code now that the test framework is managing `g_rpc_node`:

```diff
--- a/src/test/rpc_tests.cpp
+++ b/src/test/rpc_tests.cpp
@@ -112,15 +112,10 @@ BOOST_AUTO_TEST_CASE(rpc_rawsign)
     std::string notsigned = r.get_str();
     std::string privkey1 = "\"KzsXybp9jX64P5ekX1KUxRQ79Jht9uzW7LorgwE65i5rWACL6LQe\"";
     std::string privkey2 = "\"Kyhdf5LuKTRx4ge69ybABsiUAWjVRK4XGxAKk2FQLp2HjGMy87Z4\"";
-    NodeContext node;
-    node.chain = interfaces::MakeChain(node);
-    node.mempool = std::move(m_node.mempool);
-    g_rpc_node = &node;
     r = CallRPC(std::string("signrawtransactionwithkey ")+notsigned+" [] "+prevout);
     BOOST_CHECK(find_value(r.get_obj(), "complete").get_bool() == false);
     r = CallRPC(std::string("signrawtransactionwithkey ")+notsigned+" ["+privkey1+","+privkey2+"] "+prevout);
     BOOST_CHECK(find_value(r.get_obj(), "complete").get_bool() == true);
-    g_rpc_node = nullptr;
 }
```

Seems to work

Reply:
Thanks. Fixed